templates/observability/index.html

```diff
@@ -355,12 +355,15 @@ <h3 class="p-heading--four">Dashboards and Alerting</h3>
   </div>
 </section>
 
-<section class="p-strip is-deep">
-  <div class="row">
-    <div class="u-fixed-width">
-      <h2 class="u-sv3">Latest Ubuntu Observability news from <a href="/blog/tag/observability">our blog</a>&nbsp;&rsaquo;</h2>
-    </div>
-  </div>
+<div class="u-fixed-width">
```

Comment:
I think this hr and div isn't required as the above section is-bordered

Reply:
Done